src/transformers/models/trocr/processing_trocr.py

```diff
@@ -20,32 +20,34 @@
 from transformers.feature_extraction_utils import FeatureExtractionMixin
 from transformers.models.roberta.tokenization_roberta import RobertaTokenizer
 from transformers.models.roberta.tokenization_roberta_fast import RobertaTokenizerFast
+from transformers.models.xlm_roberta.tokenization_xlm_roberta import XLMRobertaTokenizer
+from transformers.models.xlm_roberta.tokenization_xlm_roberta_fast import XLMRobertaTokenizerFast
 
-from ..auto.feature_extraction_auto import AutoFeatureExtractor
+from transformers import AutoTokenizer, AutoFeatureExtractor
 
 
 class TrOCRProcessor:
     r"""
     Constructs a TrOCR processor which wraps a vision feature extractor and a TrOCR tokenizer into a single processor.
 
-    [`TrOCRProcessor`] offers all the functionalities of [`AutoFeatureExtractor`] and [`RobertaTokenizer`]. See the
+    [`TrOCRProcessor`] offers all the functionalities of [`AutoFeatureExtractor`] and [`RobertaTokenizer`/`XLMRobertaTokenizer`]. See the
     [`~TrOCRProcessor.__call__`] and [`~TrOCRProcessor.decode`] for more information.
 
     Args:
         feature_extractor ([`AutoFeatureExtractor`]):
             An instance of [`AutoFeatureExtractor`]. The feature extractor is a required input.
-        tokenizer ([`RobertaTokenizer`]):
-            An instance of [`RobertaTokenizer`]. The tokenizer is a required input.
+        tokenizer ([`RobertaTokenizer`/`XLMRobertaTokenizer`]):
+            An instance of [`RobertaTokenizer`/`XLMRobertaTokenizer`]. The tokenizer is a required input.
```

Comment:
Could you also update the docstring of the feature extractor above, to indicate it should be an instance of [`ViTFeatureExtractor`/`DeiTFeatureExtractor`] rather than `AutoFeatureExtractor`?

Internally, the processor will use the Auto API to load it, but users should pass one that is an instance of `ViTFeatureExtractor`/`DeiTFeatureExtractor`.

Reply:
Sure